jazz_test-lambda/index.js

```diff
@@ -0,0 +1,122 @@
+// =========================================================================
+// Copyright ©  2017 T-Mobile USA, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =========================================================================
+
+/**
+Nodejs Template Project
+@author:
+@version: 1.0
+ **/
+
+const errorHandlerModule = require("./components/error-handler.js"); //Import the error codes module.
+const responseObj = require("./components/response.js"); //Import the response module.
+const configObj = require("./components/config.js"); //Import the environment data.
+const logger = require("./components/logger.js"); //Import the logging module.
+const utils = require("./components/utils.js");
+const aws = require('aws-sdk');
+const validateARN = utils.validateARN;
+const execStatus = utils.execStatus();
+
+var handler = (event, context, cb) => {
+
+  'use strict';
+  //Initializations
+  var errorHandler = errorHandlerModule();
+  var config = configObj(event);
+  logger.init(event, context);
+  var awsRegion;
+  try {
+    var testResponse = {
```

Comment:
While I understand your intent behind this variable name, it is confusing as someone trying to read the code below. How about you call it `response` or `responseObj` since the service itself is called test-lambda?

Reply:
done